src/lib/settings/getDefaultRange.js

```diff
@@ -11,6 +18,14 @@ function calcBaseRange(s, width, height, reverse, percentRange) {
 	return reverse === true ? [max, min] : [min, max];
 }
 
+/**
+ * @param {'x'|'y'|'z'|'r'} s
+ * @param {number} width
+ * @param {number} height
+ * @param {boolean} reverse
+ * @param {[number, number]|Function|null} range
+ * @param {boolean} percentRange
```

Comment:
We should probably add a return description to the type like in the above comment